packages/orion/src/Layout/Layout.stories.js

```diff
@@ -58,7 +58,7 @@ export const basic = () => (
       <Layout.Topbar.Divider />
       <Layout.UserProfile
         name={text('Name', 'Mark Weiser', 'UserProfile')}
-        label={text('Label', 'In Loco', 'UserProfile')}
+        label={text('Label', 'Incognia', 'UserProfile')}
         email={text('Email', 'mark.weiser@inloco.com.br', 'UserProfile')}
```

Comment:
Já que estamos mudando pra Incognia, que tal mudar os emails tbm? :P 

```suggestion
        email={text('Email', 'mark.weiser@incognia.com', 'UserProfile')}
```